src/Websites/Websites/ChangeLog.md

```diff
@@ -18,7 +18,11 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Added support for App Service Managed certificates
+	-New Cmdlets
+		-New-AzWebAppCertificate
+		-Remove-AzWebAppCertificate
```

Comment:
```suggestion
* Added support for App Service Managed certificates
    - New Cmdlets
        - New-AzWebAppCertificate
        - Remove-AzWebAppCertificate
```
Pls be careful with markdown grammer. BTW you can preview it here
![image](https://user-images.githubusercontent.com/11371776/103731805-f97fa600-5020-11eb-9489-c65526dc5599.png)